### PR TITLE
animation class

### PR DESCRIPTION
**Pull request by Demigod916**
animation class for ox_lib to easily start and stop animations/scenarios for any ped.

included the [docs](https://github.com/overextended/overextended.github.io/pull/150) as well

### CODE REVIEW

**Comment by Demigod916** [comment]
happy to make any requested changes or additions

File: imports/anim/client.lua [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---@class AnimProperties
+---@field ped number
+---@field dict string
+---@field anim string
+---@field options table
```
Comment (thelindat) [inline imports/anim/client.lua:5]: This needs to be properly defined, not just a generic table.

**Review by thelindat — COMMENTED** [review]
I'm not sure how much time this really saves, but reviewed anyway.

File: imports/anim/client.lua (hunk 1 above)
Comment (Demigod916) [reply imports/anim/client.lua]: this as well as all your other comments should be resolved in latest commit

**Pull request closed without merging**